### DISCUSSION TITLE
Apply Single Stock to all SKUs

### DISCUSSION THREAD

**Discussion by jayasth** [saleor/saleor#7667] [Q&A]
Just wondering what do these options do?

![price](https://user-images.githubusercontent.com/6062614/126475778-305ddf01-2498-4eac-982a-56fc715c8a64.png)
![stock](https://user-images.githubusercontent.com/6062614/126476666-19cf76f2-71dd-4662-b3c2-b9cbb9788304.png)

When I am creating a new product with variants, it gives me these options to "Apply single price to all SKUs" and "Apply single stock to all SKUs" but I have to enter the individual price and quantity on the variants. 

(I have some clothing products, the size can be resized/altered physically before delivery, using 1 quantity for all size variants would have been perfect setting for my situation, but it's not working.)

**Comment by karolkielecki** [comment]
@jayasth Hi there! The `Apply single price to all SKUs` option allows you to assign prices for specific channels you have previously created. You can apply specific prices based on attributes and their variants for specific channels by selecting the Apply unique prices by attribute to each SKU` option. 

The `Apply single stock to all SKUs` option allows you to assign available quantity for specific warehouses you have previously created in `configuration` of the dashboard. Your screenshot is showing that you haven't created any warehouses where you would like to keep your stock. 

What do you mean by `1 quantity`? What exactly it's not working? Can you provide more screenshots?
